lte/gateway/c/core/oai/test/spgw_task/test_spgw_state_converter.cpp

```diff
@@ -10,12 +10,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#include <lte/gateway/c/core/oai/tasks/sgw/spgw_state_manager.h>
 #include <string.h>
 #include <gtest/gtest.h>
+#include <cstdio>
+#include <cstdlib>
 
+#include "ie_to_bytes.h"
```

Comment:
can we enclose this (and the other plain C included files) on `extern "C"` directive?